### ISSUE TITLE
Issue with PHP 7.1 stops installation

### ISSUE THREAD

**Issue by mikeerickson** [laravel/valet#219]
@adamwathan 

Trying to install valet with PHP71 and fails :-( Is there some black magic somewhere to get this working?

```
✘  ~/.composer/vendor/laravel/valet 
 $ valet --version
Laravel Valet version 1.1.22
 ~/.composer/vendor/laravel/valet 
 $ valet install
[php70] is not installed, installing it now via Brew... 🍻
==> Installing php70 from homebrew/php
Error: Cannot install homebrew/php/php70 because conflicting formulae are installed.

  php71: because different php versions install the same binaries.

Please `brew unlink php71` before continuing.

Unlinking removes a formula's symlinks from /usr/local. You can
link the formula again after the install finishes. You can --force this
install, but the build may fail or cause obscure side-effects in the
resulting software.
```

**Comment by matthijs110** [comment]
Same here

**Comment by matthijs110** [comment]
I fixed it by changing the following:

https://github.com/matthijs110/valet/blob/master/cli/Valet/PhpFpm.php#L43

**Comment by hotmeteor** [comment]
That didn't even work for me. I had to comment out that entire block of checks just to get it installed, as I knew I had `php71` installed. Pretty weird.

**Comment by sebastiandedeyne** [comment]
Ran into the same issue, this PR fixes it for me: https://github.com/laravel/valet/pull/222

**Comment by Lavoaster** [comment]
@mikeerickson This is fixed as of #222 being merged.

**Comment by mikeerickson** [comment]
@Lavoaster Thanks, I will give it a whirl later this afternoon.
